sqlalchemy_bigquery/base.py

```diff
@@ -655,25 +656,79 @@ def get_column_specification(self, column, **kwargs):
 
     def post_create_table(self, table):
         bq_opts = table.dialect_options["bigquery"]
-        opts = []
 
-        if ("description" in bq_opts) or table.comment:
-            description = process_string_literal(
-                bq_opts.get("description", table.comment)
+        partition_by_or_cluster_by_or_options = []
+
+        if "partitioning" in bq_opts:
+            partition_expr = bq_opts.get("partitioning")
+
+            assert isinstance(
```

Comment:
Please replace the `assert` statement with some other form of check.

`assert` statements are typically used in debugging as development happens OR in test statements as part of a test suite. They should not be used in production code because if the admin turns off debug mode in Python then all assert statements, such as this one, are skipped and the checks that they do will no longer be performed.

For some more details on this phenomena, see this article:

https://realpython.com/python-assert-statement/#using-assert-for-data-processing-and-validation

Reply:
Ok, got it. I will change this into standard `if` / `raise` statement.

Reply:
Thanks for sharing the article!